README.md

```diff
@@ -14,7 +14,7 @@ __DC/OS SDK__ is a collection of tools, libraries, and documentation for easy in
 
 ![Status](https://img.shields.io/badge/Status-Alpha-BF97F0.svg?style=flat-square)
```

Comment:
I think the status shield should also be changed?

Reply:
Yes.  That's what I want to remove.  How about `beta` @triclambert ?  Beta says "progressing toward a 1.0 release" to me.  Either that or we drop the shield and say:

```
DC/OS SDK is currently in beta.  It is progressing toward a 1.0 release, but is still under heavy development.
```

Reply:
Done.